testsuite/MDAnalysisTests/coordinates/test_trc.py

```diff
@@ -0,0 +1,189 @@
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
+from numpy.testing import assert_equal
+
+import MDAnalysis as mda
+from MDAnalysis.coordinates.TRC import TRCReader
+from MDAnalysisTests.datafiles import TRC_PDB_VAC, TRC_TRAJ1_VAC, TRC_TRAJ2_VAC
+from MDAnalysisTests.datafiles import TRC_CLUSTER_VAC
+from MDAnalysisTests.datafiles import TRC_PDB_SOLV, TRC_TRAJ_SOLV
+
+
+class TestTRCReaderVacuumBox:
+    @pytest.fixture(scope='class')
+    def TRC_U(self):
+        return mda.Universe(TRC_PDB_VAC, [TRC_TRAJ1_VAC, TRC_TRAJ2_VAC],
+                            continuous=True)
+
+    def test_initial_frame_is_0(self, TRC_U):
+        assert_equal(TRC_U.trajectory.ts.frame, 0,
+                     "The initial frame is not 0 but {0}".format(
+                         TRC_U.trajectory.ts.frame))
+
+    def test_trc_positions(self, TRC_U):
+        # first frame first particle
+        TRC_U.trajectory[0]
+        assert_allclose(TRC_U.atoms.positions[0],
+                        [2.19782507, 24.65064345, 29.39783426])
+        # fith frame first particle
+        TRC_U.trajectory[4]
+        assert_allclose(TRC_U.atoms.positions[0],
+                        [0.37026654, 22.78805010, 3.69695262])
+
+    def test_trc_dimensions(self, TRC_U):
+        assert TRC_U.trajectory[0].dimensions is None
+
+    def test_trc_n_frames(self, TRC_U):
+        assert len(TRC_U.trajectory) == 6
+        assert (TRC_U.trajectory.n_frames) == 6
+
+    def test_trc_n_atoms(self, TRC_U):
+        assert (TRC_U.trajectory.n_atoms) == 73
+
+    def test_trc_frame(self, TRC_U):
+        assert TRC_U.trajectory[0].frame == 0
+        assert TRC_U.trajectory[4].frame == 4
+
+    def test_trc_time(self, TRC_U):
+        assert TRC_U.trajectory[0].time == 0
+        assert TRC_U.trajectory[4].time == 80
+
+    def test_trc_dt(self, TRC_U):
+        time_array = np.array([ts.time for ts in TRC_U.trajectory])
+        assert_equal(time_array, np.arange(6)*20)
```

Comment:
For any floating point number comparisons such as coordinates, use [assert_allclose(value, reference)](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html).

Reply:
Yes, I changed that